versions.txt

```diff
@@ -1,4 +1,4 @@
 # Format:
 # module:released-version:current-version
 
-google-cloud-spanner-hibernate:1.5.3:1.5.4-SNAPSHOT
\ No newline at end of file
+google-cloud-spanner-hibernate:1.5.4:1.5.4
```

Comment:
@ddixit14 `google-cloud-spanner-hibernate:1.5.4:1.5.4` this does not seem good.

Reply:
I know this is weird, but this is how release-please deals with non-SNAPSHOT version bumps. So if we are to release "a.b.x" version, the "released-version:current-version" format followed in versions.txt is going to be " a.b.x : a.b.x "
I didn't know this but looked at a couple of SNAPSHOT v/s non-SNAPSHOT release-please PRs and came to this conclusion. For example, [this](https://github.com/googleapis/java-eventarc/pull/196/files).

Reply:
This actually made me think about how will we differentiate between SNAPSHOT version release PRs v/s non-SNAPSHOT version release PRs. The thing is, once our "non-SNAPSHOT" version release goes through, release-please automatically opens up a new "SNAPSHOT" version release PR. This "SNAPSHOT" release PR updates the pom.xml's and versions.txt of our project (and never CHANGELOG.md -> this is always changed by "non-SNAPSHOT" version release PR).